src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -683,6 +716,17 @@ class SceneryWindow final : public Window
             const auto lastTabIndex = GetMaxTabCountInARow() == MaxTabsPerRow ? MaxTabsPerRow - 1 : _tabEntries.size() - 1;
             const auto lastTabWidget = &widgets[WIDX_SCENERY_TAB_1 + lastTabIndex];
             windowWidth = std::max<int32_t>(windowWidth, lastTabWidget->right + 3);
+
+            for (size_t index = 0; index < _tabEntries.size(); index++)
+            {
+                if (!_tabEntries[index].IsAll())
+                    continue;
+
+                auto searchTabWidget = &widgets[WIDX_SCENERY_TAB_1 + index];
+                searchTabWidget->left = windowWidth - TabWidth - 11;
```

Comment:
this looks odd. Shouldn't this have some combination of index in the calculation?

Reply:
index is left out because this is supposed to shift the tab position dependent on the right edge of the window. This is to leave a gap between this tab and the remaining ones when there is only one row of scenery tabs.